app/Services/ConfigurationService.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\Setting;
+
+class ConfigurationService
+{
+    public function getThresholdValue()
+    {
+        $ThresholdValues = Setting::query()
+            ->where('setting_key', 'employee_earning_threshold')
+            ->orWhere('setting_key', 'contract_end_date_threshold')
```

Comment:
We can use [`whereIn`](https://laravel.com/docs/11.x/queries#additional-where-clauses) here
```suggestion
            ->whereIn('setting_key', $settingKeys)
```

Reply:
> We can use [`whereIn`](https://laravel.com/docs/11.x/queries#additional-where-clauses) here

Ok, Sure